<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>
<title>VecDynUser&amp;CuratorGuide</title><link href='https://fonts.loli.net/css?family=Open+Sans:400italic,700italic,700,400&subset=latin,latin-ext' rel='stylesheet' type='text/css' /><style type='text/css'>html {overflow-x: initial !important;}:root { --bg-color:#ffffff; --text-color:#333333; --select-text-bg-color:#B5D6FC; --select-text-font-color:auto; --monospace:"Lucida Console",Consolas,"Courier",monospace; }
html { font-size: 14px; background-color: var(--bg-color); color: var(--text-color); font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; -webkit-font-smoothing: antialiased; }
body { margin: 0px; padding: 0px; height: auto; bottom: 0px; top: 0px; left: 0px; right: 0px; font-size: 1rem; line-height: 1.42857; overflow-x: hidden; background: inherit; }
iframe { margin: auto; }
a.url { word-break: break-all; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { text-shadow: none; background: var(--select-text-bg-color); color: var(--select-text-font-color); }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; white-space: normal; overflow-x: visible; }
#write.first-line-indent p { text-indent: 2em; }
#write.first-line-indent li p, #write.first-line-indent p * { text-indent: 0px; }
#write.first-line-indent li { margin-left: 2em; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
.typora-export .footnote-line, .typora-export p { white-space: pre-wrap; }
@media screen and (max-width: 500px) {
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
#write li > figure:first-child { margin-top: -20px; }
#write ol, #write ul { position: relative; }
img { max-width: 100%; vertical-align: middle; }
button, input, select, textarea { color: inherit; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; font-size: inherit; line-height: inherit; font-family: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
*, ::after, ::before { box-sizing: border-box; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p, #write pre { width: inherit; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p { position: relative; }
h1, h2, h3, h4, h5, h6 { break-after: avoid-page; break-inside: avoid; orphans: 2; }
p { orphans: 4; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
.md-math-block, .md-rawblock, h1, h2, h3, h4, h5, h6, p { margin-top: 1rem; margin-bottom: 1rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: 700; font-style: italic; }
a { cursor: pointer; }
sup.md-footnote { padding: 2px 4px; background-color: rgba(238, 238, 238, 0.7); color: rgb(85, 85, 85); border-radius: 4px; cursor: pointer; }
sup.md-footnote a, sup.md-footnote a:hover { color: inherit; text-transform: inherit; text-decoration: inherit; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; }
figure { overflow-x: auto; margin: 1.2em 0px; max-width: calc(100% + 16px); padding: 0px; }
figure > table { margin: 0px !important; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 80px; }
.CodeMirror-gutters { border-right: 0px; background-color: inherit; }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
#write pre { white-space: pre-wrap; }
#write.fences-no-line-wrapping pre { white-space: pre; }
#write pre.ty-contain-cm { white-space: normal; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background: inherit; position: relative !important; }
.md-diagram-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
#write .md-fences.mock-cm { white-space: pre-wrap; }
.md-fences.md-fences-with-lineno { padding-left: 0px; }
#write.fences-no-line-wrapping .md-fences.mock-cm { white-space: pre; overflow-x: auto; }
.md-fences.mock-cm.md-fences-with-lineno { padding-left: 8px; }
.CodeMirror-line, twitterwidget { break-inside: avoid; }
.footnotes { opacity: 0.8; font-size: 0.9rem; margin-top: 1em; margin-bottom: 1em; }
.footnotes + .footnotes { margin-top: 0px; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; background: 0px 0px; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; -webkit-tap-highlight-color: transparent; line-height: normal; font-weight: 400; text-align: left; box-sizing: content-box; direction: ltr; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li .mathjax-block, li p { margin: 0.5rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child, li > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; white-space: normal; }
#write .footnote-line { white-space: pre-wrap; }
@media print {
  body, html { border: 1px solid transparent; height: 99%; break-after: avoid; break-before: avoid; }
  #write { margin-top: 0px; border-color: transparent !important; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  html.blink-to-pdf { font-size: 13px; }
  .typora-export #write { padding-left: 1cm; padding-right: 1cm; padding-bottom: 0px; break-after: avoid; }
  .typora-export #write::after { height: 0px; }
  @page { margin: 20mm 0px; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 0.8rem; white-space: pre-wrap; background: rgb(204, 204, 204); display: block; overflow-x: hidden; }
p > img:only-child { display: block; margin: auto; }
p > .md-image:only-child { display: inline-block; width: 100%; text-align: center; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.md-math-block { width: 100%; }
.md-math-block:not(:empty)::after { display: none; }
[contenteditable="true"]:active, [contenteditable="true"]:focus { outline: 0px; box-shadow: none; }
.md-task-list-item { position: relative; list-style-type: none; }
.task-list-item.md-task-list-item { padding-left: 0px; }
.md-task-list-item > input { position: absolute; top: 0px; left: 0px; margin-left: -1.2em; margin-top: calc(1em - 10px); }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc-content::after, .md-toc::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); }
.md-toc-item a { text-decoration: none; }
.md-toc-inner:hover { text-decoration: underline; }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: 700; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) {
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
code, pre, samp, tt { font-family: var(--monospace); }
kbd { margin: 0px 0.1em; padding: 0.1em 0.6em; font-size: 0.8em; color: rgb(36, 39, 41); background: rgb(255, 255, 255); border: 1px solid rgb(173, 179, 185); border-radius: 3px; box-shadow: rgba(12, 13, 14, 0.2) 0px 1px 0px, rgb(255, 255, 255) 0px 0px 0px 2px inset; white-space: nowrap; vertical-align: middle; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: var(--monospace); }
code { text-align: left; vertical-align: initial; }
a.md-print-anchor { white-space: pre !important; border-width: initial !important; border-style: none !important; border-color: initial !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: 0px !important; background: 0px 0px !important; text-decoration: initial !important; text-shadow: initial !important; }
.md-inline-math .MathJax_SVG .noError { display: none !important; }
.md-math-block .MathJax_SVG_Display { text-align: center; margin: 0px; position: relative; text-indent: 0px; max-width: none; max-height: none; min-height: 0px; min-width: 100%; width: auto; overflow-y: hidden; display: block !important; }
.MathJax_SVG_Display, .md-inline-math .MathJax_SVG_Display { width: auto; margin: inherit; display: inline-block !important; }
.MathJax_SVG .MJX-monospace { font-family: var(--monospace); }
.MathJax_SVG .MJX-sans-serif { font-family: sans-serif; }
.MathJax_SVG { display: inline; font-style: normal; font-weight: 400; line-height: normal; zoom: 90%; text-indent: 0px; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0px; min-height: 0px; border: 0px; padding: 0px; margin: 0px; }
.MathJax_SVG * { transition: none; }
.MathJax_SVG_Display svg { vertical-align: middle !important; margin-bottom: 0px !important; }
.os-windows.monocolor-emoji .md-emoji { font-family: "Segoe UI Symbol", sans-serif; }
.md-diagram-panel > svg { max-width: 100%; }
[lang="mermaid"] svg, [lang="flow"] svg { max-width: 100%; }
[lang="mermaid"] .node text { font-size: 1rem; }
table tr th { border-bottom: 0px; }
video { max-width: 100%; display: block; margin: 0px auto; }
iframe { max-width: 100%; width: 100%; border: none; }
.highlight td, .highlight tr { border: 0px; }


:root { --side-bar-bg-color: #fafafa; --control-text-color: #777; }
@font-face { font-family: "Open Sans"; font-style: normal; font-weight: normal; src: local("Open Sans Regular"), url("./github/400.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: italic; font-weight: normal; src: local("Open Sans Italic"), url("./github/400i.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: normal; font-weight: bold; src: local("Open Sans Bold"), url("./github/700.woff") format("woff"); }
@font-face { font-family: "Open Sans"; font-style: italic; font-weight: bold; src: local("Open Sans Bold Italic"), url("./github/700i.woff") format("woff"); }
html { font-size: 16px; }
body { font-family: "Open Sans", "Clear Sans", "Helvetica Neue", Helvetica, Arial, sans-serif; color: rgb(51, 51, 51); line-height: 1.6; }
#write { max-width: 860px; margin: 0px auto; padding: 30px 30px 100px; }
#write > ul:first-child, #write > ol:first-child { margin-top: 30px; }
a { color: rgb(65, 131, 196); }
h1, h2, h3, h4, h5, h6 { position: relative; margin-top: 1rem; margin-bottom: 1rem; font-weight: bold; line-height: 1.4; cursor: text; }
h1:hover a.anchor, h2:hover a.anchor, h3:hover a.anchor, h4:hover a.anchor, h5:hover a.anchor, h6:hover a.anchor { text-decoration: none; }
h1 tt, h1 code { font-size: inherit; }
h2 tt, h2 code { font-size: inherit; }
h3 tt, h3 code { font-size: inherit; }
h4 tt, h4 code { font-size: inherit; }
h5 tt, h5 code { font-size: inherit; }
h6 tt, h6 code { font-size: inherit; }
h1 { padding-bottom: 0.3em; font-size: 2.25em; line-height: 1.2; border-bottom: 1px solid rgb(238, 238, 238); }
h2 { padding-bottom: 0.3em; font-size: 1.75em; line-height: 1.225; border-bottom: 1px solid rgb(238, 238, 238); }
h3 { font-size: 1.5em; line-height: 1.43; }
h4 { font-size: 1.25em; }
h5 { font-size: 1em; }
h6 { font-size: 1em; color: rgb(119, 119, 119); }
p, blockquote, ul, ol, dl, table { margin: 0.8em 0px; }
li > ol, li > ul { margin: 0px; }
hr { height: 2px; padding: 0px; margin: 16px 0px; background-color: rgb(231, 231, 231); border: 0px none; overflow: hidden; box-sizing: content-box; }
li p.first { display: inline-block; }
ul, ol { padding-left: 30px; }
ul:first-child, ol:first-child { margin-top: 0px; }
ul:last-child, ol:last-child { margin-bottom: 0px; }
blockquote { border-left: 4px solid rgb(223, 226, 229); padding: 0px 15px; color: rgb(119, 119, 119); }
blockquote blockquote { padding-right: 0px; }
table { padding: 0px; word-break: initial; }
table tr { border-top: 1px solid rgb(223, 226, 229); margin: 0px; padding: 0px; }
table tr:nth-child(2n), thead { background-color: rgb(248, 248, 248); }
table tr th { font-weight: bold; border-width: 1px 1px 0px; border-top-style: solid; border-right-style: solid; border-left-style: solid; border-top-color: rgb(223, 226, 229); border-right-color: rgb(223, 226, 229); border-left-color: rgb(223, 226, 229); border-image: initial; border-bottom-style: initial; border-bottom-color: initial; text-align: left; margin: 0px; padding: 6px 13px; }
table tr td { border: 1px solid rgb(223, 226, 229); text-align: left; margin: 0px; padding: 6px 13px; }
table tr th:first-child, table tr td:first-child { margin-top: 0px; }
table tr th:last-child, table tr td:last-child { margin-bottom: 0px; }
.CodeMirror-lines { padding-left: 4px; }
.code-tooltip { box-shadow: rgba(0, 28, 36, 0.3) 0px 1px 1px 0px; border-top: 1px solid rgb(238, 242, 242); }
.md-fences, code, tt { border: 1px solid rgb(231, 234, 237); background-color: rgb(248, 248, 248); border-radius: 3px; padding: 2px 4px 0px; font-size: 0.9em; }
code { background-color: rgb(243, 244, 244); padding: 0px 4px 2px; }
.md-fences { margin-bottom: 15px; margin-top: 15px; padding: 8px 1em 6px; }
.md-task-list-item > input { margin-left: -1.3em; }
@media print {
  html { font-size: 13px; }
  table, pre { break-inside: avoid; }
  pre { word-wrap: break-word; }
}
.md-fences { background-color: rgb(248, 248, 248); }
#write pre.md-meta-block { padding: 1rem; font-size: 85%; line-height: 1.45; background-color: rgb(247, 247, 247); border: 0px; border-radius: 3px; color: rgb(119, 119, 119); margin-top: 0px !important; }
.mathjax-block > .code-tooltip { bottom: 0.375rem; }
.md-mathjax-midline { background: rgb(250, 250, 250); }
#write > h3.md-focus::before { left: -1.5625rem; top: 0.375rem; }
#write > h4.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
#write > h5.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
#write > h6.md-focus::before { left: -1.5625rem; top: 0.285714rem; }
.md-image > .md-meta { border-radius: 3px; padding: 2px 0px 0px 4px; font-size: 0.9em; color: inherit; }
.md-tag { color: rgb(167, 167, 167); opacity: 1; }
.md-toc { margin-top: 20px; padding-bottom: 20px; }
.sidebar-tabs { border-bottom: none; }
#typora-quick-open { border: 1px solid rgb(221, 221, 221); background-color: rgb(248, 248, 248); }
#typora-quick-open-item { background-color: rgb(250, 250, 250); border-color: rgb(254, 254, 254) rgb(229, 229, 229) rgb(229, 229, 229) rgb(238, 238, 238); border-style: solid; border-width: 1px; }
.on-focus-mode blockquote { border-left-color: rgba(85, 85, 85, 0.12); }
header, .context-menu, .megamenu-content, footer { font-family: "Segoe UI", Arial, sans-serif; }
.file-node-content:hover .file-node-icon, .file-node-content:hover .file-node-open-state { visibility: visible; }
.mac-seamless-mode #typora-sidebar { background-color: var(--side-bar-bg-color); }
.md-lang { color: rgb(180, 101, 77); }
.html-for-mac .context-menu { --item-hover-bg-color: #E6F0FE; }
#md-notification .btn { border: 0px; }
.dropdown-menu .divider { border-color: rgb(229, 229, 229); }





 .typora-export p, .typora-export .footnote-line {white-space: normal;} 
</style>
</head>
<body class='typora-export os-windows' >
<div  id='write'  class = 'is-node'><p>Global <strong>Vec</strong>tor Population <strong>Dyn</strong>amics Database (VecDyn) User Guide</p><div class='md-toc' mdtype='toc'><p class="md-toc-content"><span class="md-toc-item md-toc-h1" data-ref="n3"><a class="md-toc-inner" href="#header-n3">Introduction</a></span><span class="md-toc-item md-toc-h2" data-ref="n5"><a class="md-toc-inner" href="#header-n5">What is VecDyn?</a></span><span class="md-toc-item md-toc-h3" data-ref="n7"><a class="md-toc-inner" href="#header-n7">Database Usage</a></span><span class="md-toc-item md-toc-h3" data-ref="n15"><a class="md-toc-inner" href="#header-n15">Overview of data collection</a></span><span class="md-toc-item md-toc-h3" data-ref="n27"><a class="md-toc-inner" href="#header-n27">Temporal referencing</a></span><span class="md-toc-item md-toc-h3" data-ref="n29"><a class="md-toc-inner" href="#header-n29">Spatial referencing</a></span><span class="md-toc-item md-toc-h2" data-ref="n42"><a class="md-toc-inner" href="#header-n42">Data Storage and Data Collection Specifications</a></span><span class="md-toc-item md-toc-h3" data-ref="n44"><a class="md-toc-inner" href="#header-n44">VecDyn Database structure (backend)</a></span><span class="md-toc-item md-toc-h3" data-ref="n56"><a class="md-toc-inner" href="#header-n56">Data collection models </a></span><span class="md-toc-item md-toc-h4" data-ref="n57"><a class="md-toc-inner" href="#header-n57">Publication Information Template</a></span><span class="md-toc-item md-toc-h4" data-ref="n160"><a class="md-toc-inner" href="#header-n160">Main Data Collection Template</a></span><span class="md-toc-item md-toc-h3" data-ref="n368"><a class="md-toc-inner" href="#header-n368">Data Standardization models / Data integrity</a></span><span class="md-toc-item md-toc-h4" data-ref="n369"><a class="md-toc-inner" href="#header-n369"><strong>Taxonomic Standardization Database</strong></a></span><span class="md-toc-item md-toc-h4" data-ref="n409"><a class="md-toc-inner" href="#header-n409"><strong>Geographic Standardization Database </strong></a></span><span class="md-toc-item md-toc-h2" data-ref="n444"><a class="md-toc-inner" href="#header-n444">Curator guidelines</a></span><span class="md-toc-item md-toc-h4" data-ref="n445"><a class="md-toc-inner" href="#header-n445">Accessing the VecDyn</a></span><span class="md-toc-item md-toc-h4" data-ref="n446"><a class="md-toc-inner" href="#header-n446">Preparing data for upload</a></span><span class="md-toc-item md-toc-h3" data-ref="n457"><a class="md-toc-inner" href="#header-n457">Adding data to the database</a></span><span class="md-toc-item md-toc-h2" data-ref="n486"><a class="md-toc-inner" href="#header-n486">User Guide</a></span><span class="md-toc-item md-toc-h3" data-ref="n487"><a class="md-toc-inner" href="#header-n487">Querying the database and downloading data</a></span><span class="md-toc-item md-toc-h3" data-ref="n489"><a class="md-toc-inner" href="#header-n489">Submitting data</a></span><span class="md-toc-item md-toc-h3" data-ref="n500"><a class="md-toc-inner" href="#header-n500">Requirements for datasets not in the VecDyn template format</a></span><span class="md-toc-item md-toc-h2" data-ref="n505"><a class="md-toc-inner" href="#header-n505">Templates, Examples &amp; Tutorials</a></span><span class="md-toc-item md-toc-h2" data-ref="n511"><a class="md-toc-inner" href="#header-n511">Issues, troubleshooting &amp; suggestions</a></span><span class="md-toc-item md-toc-h3" data-ref="n513"><a class="md-toc-inner" href="#header-n513">Known issues</a></span><span class="md-toc-item md-toc-h2" data-ref="n574"><a class="md-toc-inner" href="#header-n574">Miscellaneous</a></span><span class="md-toc-item md-toc-h2" data-ref="n577"><a class="md-toc-inner" href="#header-n577">Credits</a></span><span class="md-toc-item md-toc-h2" data-ref="n579"><a class="md-toc-inner" href="#header-n579">Contact</a></span><span class="md-toc-item md-toc-h2" data-ref="n581"><a class="md-toc-inner" href="#header-n581">Citing  VecDyn</a></span><span class="md-toc-item md-toc-h2" data-ref="n583"><a class="md-toc-inner" href="#header-n583">References</a></span></p></div><h1><a name='header-n3' class='md-header-anchor '></a>Introduction</h1><p>Welcome to VectorBiTE’s VecDyn Database User and Curator guidelines. This document provides details and instructions on managing the database and web app including usage e.g. querying  (extracting data) and adding new data. </p><h2><a name='header-n5' class='md-header-anchor '></a>What is VecDyn?</h2><p>VecDyn is a global database for spatially and temporally explicit presence-absence and abundance data. We accept and distribute data for animal and plant disease vectors. </p><h3><a name='header-n7' class='md-header-anchor '></a>Database Usage</h3><ul><li><strong>Abundance data</strong>: Tracking populations over time and space</li><li><strong>Presence absence data</strong>: Tracking species presence / absence over time and space</li><li><strong>Point location data</strong>: Tracking Geo-point locations of where a species was observed. </li></ul><h3><a name='header-n15' class='md-header-anchor '></a>Overview of data collection</h3><ul><li><strong>What</strong> – description of the sample collected – (e.g. species, count)</li><li><strong>When</strong> –  temporal information about time of collection. </li><li><strong>Where</strong> – geographical information about where the collection took place. </li><li><strong>How</strong> – operational information about how the collections were made. </li><li><strong>Who</strong>  – who has responsibility for the collection</li></ul><h3><a name='header-n27' class='md-header-anchor '></a>Temporal referencing</h3><p>Temporal referencing in the VecDyn database has been kept as straightforward as possible. Collection fields are set up to gather information on the date-time a collection was started and the date-time a collection ended.  Using this information it is easy to calculate information like the duration a trap was set or to calculate the duration of other relevant sampling methods like ecological belt transects or line transects. </p><h3><a name='header-n29' class='md-header-anchor '></a>Spatial referencing</h3><p>in order to facilitate mapping and geo-spatial analysis, spatial referencing has been set up to track general information about a study site e.g. country, region, municipality,  and also specific information about a sample site, where Global Positioning System (GPS) information can be inputted for each collection site. The following figure demonstrates the spatial information that can be captured per study. </p><p>Make assessments about sampling effort </p><p>Decide whether data it is representative of a region</p><p>Use to overlay against environmental and demographic covariates</p><p>&nbsp;</p><p><img src='Images/sample-site-1.png' alt='' referrerPolicy='no-referrer' /></p><p>&nbsp;</p><p>&nbsp;</p><p><img src='Images/sample-site-2.png' alt='' referrerPolicy='no-referrer' /></p><p>&nbsp;</p><p>&nbsp;</p><h2><a name='header-n42' class='md-header-anchor '></a>Data Storage and Data Collection Specifications</h2><p>The following section provides an overview of the database structure, describes all the fields in the VecDyn database and provides details on the rational for their requirement.  </p><h3><a name='header-n44' class='md-header-anchor '></a>VecDyn Database structure (backend)</h3><p>The following image shows the database schema, this describes where the data is stored in the database, relationships between tables and the database structure.</p><p><img src='Images/erdplus-diagram.png' alt='' referrerPolicy='no-referrer' /></p><p>&nbsp;</p><p>&nbsp;</p><p>The database back-end consists of five tables:</p><p>The <strong>Publication Information Table</strong>  captures general information about the dataset e.g. titles,  descriptions,  who collected the data and data usage rights. </p><p>The <strong>Study Meta Data Table</strong> captures information describing what was collected or observed, how this was achieved,  and general details about where sampling took place. Data in this table captures information that tends to be repeated in population abundance datasets, therefore storing this information once saves storage space improving the scalability of the database. </p><p>The <strong>Time Series Data Table</strong> captures all the information required to produce a Time-Series. Each row should represent a separate observation, at a particular point in time and provide a value for the entity sampled. This table can also capture detailed temporal and spatial information about a sample/observation site e.g. specific coordinates of a trap, environmental characteristics or specific weather conditions at the time of sampling. </p><p>The <strong>Taxonomic Information Table</strong>  is a database table which is used to standardise all  taxonomic information uploaded into the database. It has been adapted from <a href='http://www.catalogueoflife.org'>The Catalogue of Life database</a>, which is the most comprehensive and authoritative global index of species currently available. This enables  front-end search (querying) facilities to use internationally recognised naming conventions thus improving the usability of the web application.   </p><p>The <strong>Geographic Database Table</strong> is used to standardize all geographic information that is stored in the database. <a href='http://www.fao.org/geonetwork/srv/en/metadata.show?id=12691'>The Global Administrative Unit Layers (GAUL) 2014 dataset</a> is a spatial database that systematizes global administrative regions with a unified coding system at country, first (e.g. departments) and second administrative levels (e.g. districts). Each Administrative unit is assigned a unique ID and is connected to a spatial polygon. Standardizing  geographic information will allow users to query data spatially or by using a structured keyword search facility. </p><p>&nbsp;</p><h3><a name='header-n56' class='md-header-anchor '></a>Data collection models </h3><h4><a name='header-n57' class='md-header-anchor '></a>Publication Information Template</h4><p>Meta-data entered into this table should be entered into the VectorBiTE web app via the data submission forms. </p><figure><table><thead><tr><th>Field Name</th><th>Required Y/N</th><th>Data format</th><th>Details</th><th>Additional Notes</th><th style='text-align:center;' >Db table</th></tr></thead><tbody><tr><td>title</td><td>Yes</td><td>String</td><td>Short title identifying the data set</td><td>E.g. “Mosquito Surveillance in Iowa”</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>collection_author</td><td>Yes</td><td>String</td><td>Name of collection author</td><td>E.g. name of a government authority, university department or individual researcher.</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>dataset_doi</td><td>No</td><td>String</td><td>Digital Object Identifier (DOI) of the the dataset</td><td>If the data set was already published</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>publication_doi</td><td>No</td><td>String</td><td>Second Digital Object Identifier (DOI)</td><td>If connected to a published article</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>description</td><td>Yes</td><td>String</td><td>A short description about the dataset and the study  objectives</td><td>E.g. ‘Long term municipal surveillance of West Nile vector population in Athens, Greece (2000-2010)”</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>url</td><td>No</td><td>String (URL)</td><td>web link to dataset source</td><td>&nbsp;</td><td style='text-align:center;' >&nbsp;</td></tr><tr><td>contact_name</td><td>Yes</td><td>String</td><td>Name of person that may be contacted with inquiries about the data</td><td>E.g. name  of the person who may be contacted with enquiries about the dataset.</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>contact_affiliation</td><td>No</td><td>String</td><td>Author/contact affiliation</td><td>If different from collection author</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>email</td><td>No</td><td>String</td><td>Contact email address</td><td>Email address of the person who may be contacted with enquiries about the dataset.</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>orchid</td><td>No</td><td>String</td><td>Open Researcher and Contributor ID</td><td>A unique code used to identify scientific/academic authors and contributors</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>dataset_license</td><td>No</td><td>String</td><td>Creative commons license identification or equivalent.</td><td>Where possible provide a creative commons license identification. See <a href='https://creativecommons.org' target='_blank' class='url'>https://creativecommons.org</a>  for more information.</td><td style='text-align:center;' >Publication information</td></tr><tr><td>data_rights</td><td>Yes</td><td>String</td><td>The data reuse policy  for the data set.</td><td>Write &#39;open&#39; if available to the public, &#39;closed&#39; If data is only available to certain groups or  &#39;embargo&#39; along with release date if under embargo.</td><td style='text-align:center;' >Publication information</td></tr><tr><td>embargo_release_date</td><td>No</td><td>Date</td><td>ISO 8601 date format (YYYY-MM-DD)</td><td>Date when the dataset will be freed from the embargo</td><td style='text-align:center;' >Publication information</td></tr></tbody></table></figure><p>&nbsp;</p><h4><a name='header-n160' class='md-header-anchor '></a>Main Data Collection Template</h4><p>The latest <a href='https://github.com/vectorbite/VectorBiteDataPlatform/blob/master/static/Documentation/VecDyn/Template%26Scripts/VecDyn_template.csv'>vecdyn data collection template</a> is available here</p><figure><table><thead><tr><th>Field Name</th><th>Required Y/N</th><th>Data format</th><th>Details</th><th>Additional Notes</th><th style='text-align:center;' >Db table</th></tr></thead><tbody><tr><td>title</td><td>Yes</td><td>String</td><td>Short title identifying the data set</td><td>E.g. “Mosquito Surveillance in Iowa”</td><td style='text-align:center;' >Publication Information</td></tr><tr><td>taxon</td><td>Yes</td><td>String</td><td>Classification of sample collected</td><td>&nbsp;</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>location_description</td><td>Yes</td><td>String</td><td>Description of study location</td><td>In order of smallest geographic subdivision e.g.  district, town, county, state, Country</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>study_collection_area</td><td>No</td><td>String</td><td>The spatial extent (area or volume) of the study.</td><td>Information supplied here should represent the total study area. A study area could contain multiple sample sites.</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>geo_datum</td><td>No</td><td>String</td><td>Geodetic datum</td><td>E.g. WGS 84</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>gps_obfuscation_info</td><td>No</td><td>String</td><td>GPS obfuscation information</td><td>If GPS data obfuscation (e.g. GPS points are intentionally offset from their actual locations) or de-resolution occurs (e.g. GPS precision is intentionally reduced) , a statement on the manner by which this occurred. “GPS locations have been truncated to 3 decimals”; “GPS locations obfuscated using N-Dispersion”;  ”No GPS de-resolution was performed”</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>species_id_method</td><td>No</td><td>String</td><td>Species identification method</td><td>A description of the methods used to identify the species e.g. “Morphological”  or  &quot;Genotype&quot;.</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>study_design</td><td>No</td><td>String</td><td>Study design methodology</td><td>Indicate if observational study i.e. prospective, retrospective, or experimental etc</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>sampling_strategy</td><td>No</td><td>String</td><td>Sampling_strategy, indicate the strategy used to select the sample</td><td>E.g. Probability sampling techniques, such as simple random sampling, systematic random sampling and stratified random sampling. Non-probability sampling techniques , such as self-selection sampling, convenience sampling, snowball sampling and purposive sampling.</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>sampling_method</td><td>No</td><td>String</td><td>Sampling apparatus e.g.trap type, observation method used to sample the entities.</td><td>E.g.    “CDC light trap” ,      “Tick drag”,      “Quadrat count”,       “BG Sentinel Trap” ,      “Pitfall trap”,       “Larval dip”,      “Johnson suction trap”,        or “Lindgren Funnel Trap”</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>sampling_protocol</td><td>No</td><td>String</td><td>How entities were sample</td><td>The sampling protocol can further describe any distinguishing procedures used during sample collection e.g. if traps were set at specific times of day, or under specific weather conditions. Furthermore, this field should be used to describe any attractants/lures used to attract entities to a trap or collection site.</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>measurement_unit</td><td>Yes</td><td>String</td><td>Description of exactly what was observed, the unit for “Value”</td><td>‘Count’, ‘Count (millions)’, ‘Harvest’, ‘Index of abundance’, ‘Index of territories’, ‘Leaf area’, ‘Mean Count’, ‘Not Specified’, ‘Percent cover’ and ‘Sample’</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>value_transform</td><td>No</td><td>String</td><td>Note if the original values have been transformed – list details of the reference value of any data transformation</td><td>E.g .Base Year, Log, Proportion</td><td style='text-align:center;' >Study Metadata</td></tr><tr><td>sample_start_date</td><td>No</td><td>ISO 8601 date format (YYYY-MM-DD)</td><td>Date the data collection started</td><td>Note this field is only required when samples /observations were made between specific time frames</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_start_time</td><td>No</td><td>ISO 8601 time format (hh:mm:ss)</td><td>Time the data collection started</td><td>Note this field is only required when samples /observations were made between specific time frames</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_end_date</td><td>Yes</td><td>ISO 8601 date format (YYYY-MM-DD)</td><td>Date the data collection ended</td><td>&nbsp;</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_end_time</td><td>No</td><td>ISO 8601 time format (hh:mm:ss)</td><td>Time the data collection ended</td><td>&nbsp;</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>value</td><td>Yes</td><td>Integer/float</td><td>The numerical amount or result from the sample collection</td><td>If presence / absence data, then values should be provided as 1 = species present &amp; 0 = species absent</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_sex</td><td>No</td><td>String</td><td>Information on the sex of the organism sampled</td><td>&nbsp;</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_stage</td><td>No</td><td>String</td><td>Information on the life stage of the organism sampled</td><td>E.g adult, egg, larva, pupa</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_location</td><td>No</td><td>String</td><td>Name of specific sample location</td><td>&nbsp;</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_collection_area</td><td>No</td><td>string</td><td>Area of sample location</td><td>If relevant (<em>e.g</em>., when collection method is transect or quadrat), in units of area or volume, the spatial coverage of the sampling unit</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_latitude_dd</td><td>No</td><td>Float</td><td>Latitude of sample area as a decimal degree Specific location of the sample</td><td>Ranges [-90,+90] for latitude (north-south measurement)</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_longitude_dd</td><td>No</td><td>Float</td><td>Longitude of sample area as a decimal degree</td><td>Ranges [-180,180] for longitude (east-west measurement)</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_environment</td><td>No</td><td>String</td><td>Additional c information relating to the environment</td><td>Should be used to describe specific information about the location / environment e.g.  forest, field , urban area; elevation.</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>additional_location_info</td><td>No</td><td>String</td><td>Additional geographic information</td><td>Should be used to describe specific information about the location e.g. near to house, near playing fields, Near to population of farm animals</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>additional_sample_info</td><td>No</td><td>String</td><td>Additional sample information</td><td>Should be used to describe specific information about conditions at the time of sampling e.g. wind speeds, temperatures, weather.</td><td style='text-align:center;' >Time Series data</td></tr><tr><td>sample_name</td><td>No</td><td>String</td><td>A human readable sample name</td><td>May exist solely for the benefit of the depositor in organizing their data, use their own internal naming conventions etc. Naming convention is not restricted, but any encoded metadata should be revealed in the other data fields. For example, you may name a sample named ‘Aphid1_StickyTrap_Jan4,’ but you will still have “Sticky Trap” listed in a Collection Method field, and “Jan 4, 2017” in the date field.</td><td style='text-align:center;' >Time Series data</td></tr></tbody></table></figure><p>&nbsp;</p><h3><a name='header-n368' class='md-header-anchor '></a>Data Standardization models / Data integrity</h3><h4><a name='header-n369' class='md-header-anchor '></a><strong>Taxonomic Standardization Database</strong></h4><p>The original <a href='http://www.catalogueoflife.org'>The Catalogue of Life database</a> field names had to be renamed for issues concerning Python keywords, the main programming language that was used to build the Web app and control the database. </p><figure><table><thead><tr><th>Original Field Name</th><th>Field Name in VecDyn Db</th></tr></thead><tbody><tr><td>taxonID</td><td>taxonomic_ID</td></tr><tr><td>kingdom</td><td>taxonomic_kingdom</td></tr><tr><td>phylum</td><td>taxonomic_phylum</td></tr><tr><td>class</td><td>taxonomic_class</td></tr><tr><td>order</td><td>taxonomic_order</td></tr><tr><td>superfamily</td><td>taxonomic_superfamily</td></tr><tr><td>genus</td><td>taxonomic_genus</td></tr><tr><td>subgenus</td><td>taxonomic_subgenus</td></tr><tr><td>specificEpithet</td><td>taxonomic_specificEpithet</td></tr><tr><td>infraspecificEpithet</td><td>taxonomic_infraspecificEpithet</td></tr><tr><td>species</td><td>taxonomic_species</td></tr></tbody></table></figure><p>&nbsp;</p><h4><a name='header-n409' class='md-header-anchor '></a><strong>Geographic Standardization Database </strong></h4><p><a href='http://www.fao.org/geonetwork/srv/en/metadata.show?id=12691'>The Global Administrative Unit Layers (GAUL) 2014 dataset</a> has been restructured in order to provide one unique observation for each individual geographic entry  or row within the data table e.g. a specific country or specific region. Each specific geographic entity has as individual conde assigned to it and shape file. </p><figure><table><thead><tr><th>Field name  in original dataset</th><th>Field name in VecDyn Db</th><th>Description</th></tr></thead><tbody><tr><td>------------------------------</td><td>ADM_CODE</td><td>unique code assigned to each administrative unit / shape file</td></tr><tr><td>ADM0_NAME</td><td>ADM0_NAME</td><td>country name</td></tr><tr><td>ADM1_NAME</td><td>ADM1_NAME</td><td>first administrative subdivision e.g. Florida</td></tr><tr><td>ADM2_NAME</td><td>ADM2_NAME</td><td>first administrative subdivision e.g. Manatee County</td></tr><tr><td>------------------------------</td><td>centroid_latitude</td><td>latitude of centroid representing centre of administrative unit</td></tr><tr><td>------------------------------</td><td>centroid_longitude</td><td>longitude centroid taking representing centre of administrative unit</td></tr></tbody></table></figure><p><img src='Images/gual-world-map.png' alt='' referrerPolicy='no-referrer' /></p><p><img src='Images/gaul-us-mex-map.png' alt='' referrerPolicy='no-referrer' /></p><p>&nbsp;</p><p>&nbsp;</p><h2><a name='header-n444' class='md-header-anchor '></a>Curator guidelines</h2><h4><a name='header-n445' class='md-header-anchor '></a>Accessing the VecDyn</h4><h4><a name='header-n446' class='md-header-anchor '></a>Preparing data for upload</h4><p><em>To prepare a dataset for the VecDyn database, follow the subsequent
guidelines</em></p><ol start='' ><li>Download the latest template by right clicking on the following <a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Template%26Scripts/VecDyn_template.csv'>link</a> and selecting ‘save.as’.   A completed <a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/vecdyn_manatee_county_aedes_aegypti.csv'>example data set can be found here</a>. This will help you to understand how to compile data in the VecDyn template format. You can also access an example <a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/ManateeCountryMosquitoMonitoring.Rmd'>R Markdown recipe</a> that provides an example of converting an existing data set into a VecDyn formatted dataset.</li><li>You can refer to the <a href='#vecdyn-data-collection-specifications'>VecDyn Data Collection Specifications</a> to guide you through the data collection and compilation process.</li><li>Note that the data upload facility (<em>temporary</em>) is only set up to process one dataset at a time, one data set compromises of one species and one main umbrella geographical location.</li></ol><p><em>Notice that all files are in plain text e.g. ‘.csv’ format, and not in Excel format or similar. This facilitates text parsing by scripts, prevents data loss/corruption, and allows for detailed comparisons of changes via version control systems.</em></p><h3><a name='header-n457' class='md-header-anchor '></a>Adding data to the database</h3><p>For testing purposes (<em>temporary</em>) first download the <a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/vecdyn_manatee_county_aedes_aegypti.csv'>example data set</a> to try out the process.</p><p>Log on and go to the and go to the <a href='http://www.vectorbyte.org'>VectorBiTE web app</a> and go to ‘My collections’. This will
only work if you have been granted access rights.</p><p><em>Note that the data upload facility (temporary) is only set up to process one dataset at a time, one data set compromises of one species and one main umbrella geographical location.</em></p><p>Click on <strong>‘Add New Collection’</strong></p><p>&nbsp;</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep1.png" width="75%" style="display: block; margin: auto;" /></p><p>&nbsp;</p><p>The first table captures general information about the data provider and the data series. A data collection may provide centralized information about one or many related datasets.</p><p>&nbsp;</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep2.png" width="75%" style="display: block; margin: auto;" /></p><p>&nbsp;</p><p>Once the collection information has been registered, you now be able to submit data sets to that collection. Click on ‘Add new data set to collection.</p><p>&nbsp;</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep3.png" width="75%" style="display: block; margin: auto;" /></p><p>Next select a taxon, it is best to search using the first box first. When you have found the taxon you are after. Hit the ‘select’ button on the aligning row.</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep4.png" width="75%" style="display: block; margin: auto;" /></p><p>Next select a geographical location, this either needs to be country or an ADM1 (e.g. State) or ADM2 (county) administrative subdivision. Again, hit the ‘select’ button on the row of your choice.</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep5.png" width="75%" style="display: block; margin: auto;" /></p><p>Next submit all the study data (metadata) and click on submit once you have completed the page.</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep6.png" width="75%" style="display: block; margin: auto;" /></p><p>Next you need to upload the complete csv. This will only upload fields related to the time series (sample data)</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep7.png" width="75%" style="display: block; margin: auto;" /></p><p>Once completed, you’ll be taken the final page where you can can verify if all the sample data is correct. If it is, click on ‘finish’ button.</p><p><img src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep9.png" width="75%" style="display: block; margin: auto;" /></p><p>However, if there is a problem you can delete or edit each entry. To delete all the entries hit ‘select’ all and scroll down to the bottom of the page and click ‘delete selected’<br/><img
src="https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Images/DataUploadStep8.png"
width="75%" style="display: block; margin: auto;" /></p><p>Note that you can also edit every part of your data after it has been submitted with the exception of taxon names and place names.</p><p>Restricted Data</p><p>&nbsp;</p><h2><a name='header-n486' class='md-header-anchor '></a>User Guide</h2><h3><a name='header-n487' class='md-header-anchor '></a>Querying the database and downloading data</h3><p>TODO</p><h3><a name='header-n489' class='md-header-anchor '></a>Submitting data</h3><p><em>If you want to submit a dataset that is not in the vecdyn format, please follow <a href='#requirements-for-data-sets-not-in-the-vecdyn-format'>Requirements for data sets not in the VecDyn format</a>. Otherwise please read below</em></p><ol start='' ><li>Download the latest template by right clicking on the following <a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Template%26Scripts/VecDyn_template.csv'>link</a> and selecting ‘save.as’.<br/>A completed <a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/vecdyn_manatee_county_aedes_aegypti.csv'>example data set can be found here</a>. This will help you to understand how to compile data in the VecDyn template format. You can also access an example <a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/ManateeCountryMosquitoMonitoring.Rmd'>R Markdown recipe</a> that provides an example of converting an existing data set into a VecDyn formatted dataset. Notice that all files are in plain text e.g. ‘.csv’ format, and not in Excel format or similar. This facilitates text parsing by scripts, prevents data loss/corruption, and allows for detailed comparisons of changes via version control
systems.</li><li>You can refer to the <a href='#vecdyn-template-field-descriptions'>VecDyn template field descriptions</a> to guide you through the data collection and compilation process.</li><li>To submit your dataset, upload and submit the dataset via the <a href='http://vectorbyte.org/default/submit_data'>upload page</a> on the VectorBiTE web app.</li><li>We’ll contact you regarding the outcome of your submission once we have had a look at it.</li></ol><h3><a name='header-n500' class='md-header-anchor '></a>Requirements for datasets not in the VecDyn template format</h3><p><em>If you are submitting a dataset not in the VecDyn format, please make
sure you supply the following information.</em></p><table>
<colgroup>
<col style="width: 19%">
<col style="width: 60%">
<col style="width: 20%">
</colgroup>
<thead>
<tr class="header">
<th>Field</th>
<th>Details</th>
<th>Additional Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Data set name</td>
<td>Short title identifying the data set</td>
<td>E.g. Mosquito Surveillance in Iowa</td>
</tr>
<tr class="even">
<td>Taxon name/s</td>
<td>Classification of samples collected</td>
<td></td>
</tr>
<tr class="odd">
<td>Country/countries</td>
<td>Country or countries where study was conducted</td>
<td></td>
</tr>
<tr class="even">
<td>Description of locations</td>
<td>Description of study locations in order of largest to smallest subdivision</td>
<td>E.g. state, county, town</td>
</tr>
<tr class="odd">
<td>Sample end date &amp; time</td>
<td>The date and time a sample was collected</td>
<td>Time is optional</td>
</tr>
<tr class="even">
<td>Value</td>
<td>The numerical amount or result from the sample collection</td>
<td></td>
</tr>
<tr class="odd">
<td>Measurement unit</td>
<td>What is the unit of measurement</td>
<td>E.g. ‘number of individuals’, ‘presence/absence’, ‘Presence only’, ‘proportion’, Percent cover</td>
</tr>
</tbody>
</table><p>## </p><h2><a name='header-n505' class='md-header-anchor '></a>Templates, Examples &amp; Tutorials</h2><p><a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Template%26Scripts/VecDyn_template.csv'>VecDyn Template</a>.</p><p><a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/ManateeCountryMosquitoMonitoring.Rmd'>Build a VecDyn Template dataframe in R Markdown</a>.</p><p><a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/vecdyn_manatee_county_aedes_aegypti.csv'>Example data set</a></p><p><a href='https://raw.githubusercontent.com/vectorbite/VectorBiteDataPlatform/master/static/Documentation/VecDyn/Examples/ManateeCountyMosquitoMonitoring/ManateeCountryMosquitoMonitoring.Rmd'>R markdown recipe: Clean a data set, transform it into the VecDyn format and produce a time
series</a></p><h2><a name='header-n511' class='md-header-anchor '></a>Issues, troubleshooting &amp; suggestions</h2><p>Any suggestions or todos with regards to the database (e.g. new columns, schema modifications etc.) can be logged as <a href='https://github.com/vectorbite/VectorBiteDataPlatform/issues'>Issues on GitHub</a>.
Issues allow for discussions among multiple users, file attachments, colour-coded labels etc.</p><h3><a name='header-n513' class='md-header-anchor '></a>Known issues</h3><p>The FAO’s GUAL data set has been restructured for VecDyn. A new column was created which represents an individual ID for each admin unit. This creates a few minor issues, since some region codes are not unique and therefore, an additional ‘b’ has been added to the end of each ADM_CODE’ which has been used in VecDyn as a ‘Primary key’. </p><p>&nbsp;</p><figure><table><thead><tr><th>ADM_CODE</th><th>ADM2_NAME</th><th>ADM1_NAME</th><th>ADM0_NAME</th></tr></thead><tbody><tr><td>48472</td><td>Administrative unit not available</td><td>Rukwa</td><td>United Republic of Tanzania</td></tr><tr><td>48472<strong>b</strong></td><td>Administrative unit not available</td><td>Mwanza</td><td>United Republic of Tanzania</td></tr><tr><td>22917</td><td>Ifelodun</td><td>Kwara</td><td>Nigeria</td></tr><tr><td>23036</td><td>Surulere</td><td>Oyo</td><td>Nigeria</td></tr><tr><td>22917<strong>b</strong></td><td>Ifelodun</td><td>Osun</td><td>Nigeria</td></tr><tr><td>23036b</td><td>Surulere</td><td>Lagos</td><td>Nigeria</td></tr><tr><td>22602</td><td>Osisioma Ngwa</td><td>Abia</td><td>Nigeria</td></tr><tr><td>22602<strong>b</strong></td><td>Ukwa West</td><td>Abia</td><td>Nigeria</td></tr><tr><td>15426</td><td>Gnral. Antonio Elizalde</td><td>Guayas</td><td>Ecuador</td></tr><tr><td>15426<strong>b</strong></td><td>Milagro</td><td>Guayas</td><td>Ecuador</td></tr></tbody></table></figure><p>&nbsp;</p><h2><a name='header-n574' class='md-header-anchor '></a>Miscellaneous</h2><p>The <code>GPDD</code> directory also contains the documentation of the <a href='http://www3.imperial.ac.uk/cpb/databases/gpdd'><strong>G</strong>lobal <strong>P</strong>opulation <strong>D</strong>ynamics <strong>D</strong>atabase</a>, a pre-existing database of population dynamics, hosted in Silwood Park, Imperial College London since 1999. </p><p>&nbsp;</p><h2><a name='header-n577' class='md-header-anchor '></a>Credits</h2><p>TODO</p><h2><a name='header-n579' class='md-header-anchor '></a>Contact</h2><p>Submitting data</p><h2><a name='header-n581' class='md-header-anchor '></a>Citing  VecDyn</h2><p>TODO</p><h2><a name='header-n583' class='md-header-anchor '></a>References</h2><p>TODO</p></div>
</body>
</html>